src/commands/help/help.ts

```diff
@@ -265,6 +265,16 @@ export async function printHelpText() {
         )} - triggers flow for execution.`,
         `[2spaces]command example: sasjs flow execute --source /local/flow.json --logFolder /local/log/folder --csvFile /local/some.csv`
       ]
+    },
+    {
+      name: 'doc',
+      title: 'doc <command>',
+      description: [
+        `Generates docs for SAS Programs / Macros / Jobs / Services for all targets and root of sasjsconfig by default. If target is provided, generated docs will be according to target.`,
```

Comment:
`Generates docs for all SAS Programs / Macros / Jobs / Services listed in the sasjsconfig.json file by default. If a target is provided, the generated docs will ignore content from the other targets.`